sale_order_general_discount/views/sale_order_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" ?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="sale_order_general_discount_form_view" model="ir.ui.view">
+        <field name="name">sale.order.general.discount.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='tax_totals_json']" position="before">
+                <table class="oe_right">
+                    <tbody>
+                        <tr>
+                            <td class="o_td_label">
+                                <label for="general_discount" />
+                            </td>
+                            <td>
+                                <field name="general_discount" nolabel="1" />
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
```

Comment:
Testing in runbut I think this would be enough:

```suggestion
                <field name="general_discount" class="oe_subtotal_footer" />
```

Result:

![image](https://user-images.githubusercontent.com/5040182/203937582-0883f882-0019-4dfd-b4ce-109500a53e5e.png)

Reply:
It looks fine, thanks.